app/views/absences/_form.html.erb

```diff
@@ -17,8 +17,8 @@
   </div>
 
   <div class="field">
-    <%= f.label :performanceset_id %>
-    <%= f.text_field :performanceset_id %>
+    <%= f.label :performance_set_id %>
```

Comment:
@dan-brewster These changes are concerning to me, if you know/think the absences controller is working in production. Related to my comments in PR #3 about the happy path forward.